os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/common/os/tizenrt/osif_tizenrt.c

```diff
@@ -815,7 +815,7 @@ bool osif_timer_start(void **pp_handle)
 	}
 
 	timer = *pp_handle;
-	if (work_queue(LPWORK, timer->work_hdl, osif_timer_wrapper, (void *) (timer), (timer->timeout * TICK_PER_SEC / 1000L)) != OK) {
+	if (work_queue(HPWORK, timer->work_hdl, osif_timer_wrapper, (void *) (timer), (timer->timeout * TICK_PER_SEC / 1000L)) != OK) {

```

Comment:
Is it okay not to change other points?
Such as, _osif_timer_restart_, _osif_timer_wrapper_, _osif_timer_delete_, _osif_timer_stop_

Reply:
I have recommited and these places are included.